services/brig/src/Brig/API.hs

```diff
@@ -190,6 +190,15 @@ sitemap o = do
     post "/i/users/blacklist" (continue addBlacklist) $
         param "email" ||| param "phone"
 
+    get "/i/users/phone-prefix" (continue getPhonePrefixes) $
```

Comment:
The usecase is going to be "look up a certain phone to figure out why it's blocked", right? Or..?

(Whatever the usecase is, maybe there should be a comment about the intended usecase.)

Reply:
also: why is it under `/i/users`?  (it's ok for constency with blacklist, just curious about the original reason.)

Reply:
the phone prefixes are related to users in some way (it blocks some phone numbers - a user's property), unlike e.g. `/i/monitoring`, which is unrelated to users.
Plural is `prefixes`, not `prefices`. Changed the paths to be plural.